guide/src/StyleGuide.js

```diff
@@ -141,7 +141,7 @@ export default React.createClass({
             main: {
                 background: "whitesmoke",
                 width: "100%",
-                margin: "50px 0px 50px 250px", 
+                margin: "50px 0px 0px 250px", 
```

Comment:
Scroll to the [bottom](http://cyverse.github.io/troposphere-ui/) there is upper margin for the top nav bar, but there is no need for the bottom margin.Thank you for being thorough